mkdocs.yml

```diff
@@ -257,7 +257,6 @@ pages:
       - Overview: sematext-enterprise/index.md
       - Licensing: sematext-enterprise/licensing.md
       - On-premises vs SaaS: sematext-enterprise/on-premises-vs-saas.md
-      - FAQ: sematext-enterprise/faq.md
```

Comment:
Why delete the FAQ?  Some of the questions on https://sematext.com/docs/sematext-enterprise/faq/ are still valid despite all the k8s stuff.

Reply:
The FAQ is now part of the guide and is more technical. I don't think it makes sense to have them in 2 places
There are just a few Q that still seem relevant:
* Can I install Sematext Enterprise if my server is behind a firewall
* What server is used for sending emails

Reply:
+1 for keeping the first one as that is something some people may wonder while exploring.  -1 for keeping the second one because that's something one thinks about only when actually setting things up.
But the point is we shouldn't remove the FAQ page all together.  For example, a good entry to add would be how STE and SC are different.  Or how does one upgrade STE.  Or what is EOL policy, and such.

Reply:
I agree. Keep the FAQ.

Reply:
@otisg there is already a separate page for STE vs SC: https://github.com/sematext/docs/blob/master/docs/sematext-enterprise/on-premises-vs-saas.md

Regarding upgrades, it is an unknown to me also.
Do we have an EOL policy?